index.html

```diff
@@ -22,7 +22,7 @@
         </a>
       </div>
       <div id="copyrightDiv">
-        © OpenStreetMap contributors © geOps MapTiles
```

Comment:
I guess we need to add Mapbox attribution. See https://www.mapbox.com/help/how-attribution-works/ for details.